File: conversions/csharp_unity/World.cs
using System;
using System.Collections.Generic;
[System.Serializable]
public class World
{
    public string Id;
    public string Name;
    public string Description;
    public string ImageUrl;
    public string ApiKey;
    public string Version;
    public string User;
    public List<int> TimeFormatEquivalents; = new List<int>();
    public List<string> TimeFormatNames; = new List<string>();
    public string TimeBasicUnit;
    public int? TimeRangeMin;
    public int? TimeRangeMax;
    public int? TimeCurrent;
}

File: languages/typescript/World.ts

export interface World {
  id: string;
  name: string;
  description?: string | null;
  imageUrl?: string | null;
  apiKey: string;
  version: string;
  user: string;
  timeFormatEquivalents?: number[] | null;
  timeFormatNames?: string[] | null;
  timeBasicUnit?: string | null;
  timeRangeMin?: number | null;
  timeRangeMax?: number | null;
  timeCurrent?: number | null;
}
